contracts/azorius/LinearERC721Voting.sol

```diff
@@ -0,0 +1,315 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity =0.8.19;
+
+import { IERC721 } from "@openzeppelin/contracts/token/ERC721/IERC721.sol";
+import { BaseVotingBasisPercent } from "./BaseVotingBasisPercent.sol";
+import { IAzorius } from "./interfaces/IAzorius.sol";
+import { BaseStrategy } from "./BaseStrategy.sol";
+
+/**
+ * Allows multiple ERC721 tokens to be registered on the strategy as governance tokens, 
+ * each with their own voting weight.
+ *
+ * Since there is no way to snapshot ERC721 holdings, each ERC721 id can vote once, reguardless
+ * of where it was when a proposal was created.
+ *
+ * Also, this uses "quorumThreshold" rather than quorumPercent, because total supply is not knowable
+ * within the IERC721 interface.  This is similar to a multisig "total signers" required, rather than
+ * a percentage of the tokens.
+ */
+contract LinearERC721Voting is BaseStrategy, BaseVotingBasisPercent {
+
+    struct GovernanceNFT {
+        bool exists;
+        uint256 weight;
+        bool isProposer;
```

Comment:
Instead of having this as a bool, could we have a `proposerThreshold` and use the weights to determine if a user has enough NFTs to be a proposer?